nexus/posts/api.py

```diff
@@ -0,0 +1,12 @@
+# Third Party Stuff
+from rest_framework import viewsets
+
+# nexus Stuff
+# from nexus.base import response
+
+from . import models, serializers
+
+
+class PostViewSet(viewsets.ModelViewSet):
```

Comment:
Please use `GenericViewSet` along with Mixins.

`ModelViewSet` would soon become outdated as your app grows.

Reply:
Okay. That would be better to scale.

Reply:
After implementing the complete API I guess we have ModelViewSet as the best option instead of using 5 mixins for List, Create, Retrieve, Update and Delete along with a GenericViewSet. Since `ModelViewSet` is giving us all these options and we can easily override when possible. We can make changes to this class if necessary in the future.